web/src/main/java/se/inera/intyg/webcert/web/integration/util/HoSPersonHelper.java

```diff
@@ -136,8 +137,23 @@ public static void enrichHoSPerson(HoSPersonal hosPerson, IntygUser user) {
         // set titel, medarbetaruppdrag, befattningar and specialiteter from user object
         hosPerson.setTitel(user.getTitel());
         hosPerson.setMedarbetarUppdrag(user.getSelectedMedarbetarUppdragNamn());
-        hosPerson.getBefattningar().addAll(user.getBefattningar());
         hosPerson.getSpecialiteter().addAll(user.getSpecialiseringar());
+        Optional.ofNullable(user.getBefattningsKoder())
```

Comment:
You are not consistent with checking if befattningskoder is null. From what I can see from the other code is that it should always pass a list and never a null.

Either remove the null check if it can never be null or be consistent :-)

See this as a comment that can affect your other PRs.